packages/contracts/bin/deploy.ts

```diff
@@ -82,10 +82,12 @@ main()
 
 function parseEnv() {
   function ensure(env, type) {
-    if (typeof process.env[env] === 'undefined')
+    if (typeof process.env[env] === 'undefined'){
```

Comment:
should be a space here. 
Can we apply a rule for that? :)

Reply:
I think I can add the `space-before-blocks` rule to check for this.

Reply:
great thx!